gridcomps/History/Sampler/MAPL_EpochSwathMod.F90

```diff
@@ -213,14 +215,15 @@ subroutine verify_epoch_equals_freq (this, frequency_from_list, swath_grid_label
          label=trim(swath_grid_label)//'.Epoch:', default=0, _RC)
     local_swath_epoch_sec = MAPL_nsecf( time_integer )
 
+    lgr => logging%get_logger('HISTORY.sampler')
```

Comment:
This is not right, but we can fix it up later.   The problem is that the "real" name of the History component is typically `GCM.HIST` so the logger you get here is not controlled by the HIST entry in logging.yaml.

In fact this is probably a major hole in how we deal with loggers, as the "full name" of a logger is not easily accessible by the end user.   You can request the current logger from MAPL generic and ... there might be a getter to give you that name.  

In any event - don't worry about it for now.